nautobot_chatops/dispatchers/base.py

```diff
@@ -1,15 +1,16 @@
 """Generic base class modeling the API for sending messages to a generic chat platform."""
-
 import logging
-
+from typing import Dict
 from django.templatetags.static import static
-
+from django.core.cache import cache
 from django.conf import settings
 
 from texttable import Texttable
 
 logger = logging.getLogger("rq.worker")
 
+DEFAULT_SESSION_CACHE_TIMEOUT = 86400
```

Comment:
Let's set this value as a `default_settings` value on the `NautobotChatOpsConfig` class, as that's more consistent with the plugin conventions.

Reply:
Can remove this from here now as it's unused.